pkg/secrethub/repo.go

```diff
@@ -107,6 +130,45 @@ func (s repoService) ListEvents(path string, subjectTypes api.AuditSubjectTypeLi
 	return events, nil
 }
 
+// EventIterator returns an iterator that retrieves all audit events for a given repo.
+//
+// Usage:
+//  iter, err := client.Repos().EventIterator(path)
+//  if err != nil {
+//  	   // Handle error
+//  }
+//  for {
+//      event, err := iter.Next()
+//	    if err == secrethub.IteratorDone {
+//	        break
+//	    }
+//	    if err != nil {
```

Comment:
Personally I feel this should be an `else if`, but I know you guys may feel differently about that.